docs/premium-features/break-the-monolith.md

```diff
@@ -100,6 +119,18 @@ This approach allows you to import your existing database structure into Amplica
 To get started with your Prisma schema upload, visit our [Import Prisma Schema](/how-to/import-prisma-schema) documentation for step-by-step instructions.
 -->
 
+## Architecture Change Confirmation
+
+Once you're satisfied with the changes, use the **Apply Plan** button to update your project's architecture.
+
+After applying changes in the Architecture tab, the "Pending Changes" section of your dashboard will reflect those updates. If everything looks good, click on the **Commit Changes & Build** button and Amplication will create a new PR to your git provider with your changes.
+
+:::note
```

Comment:
I think this node should be displayed earlier in the page (I think that right before the "Architecture Change Confirmation" part